include/rdma/fabric.h

```diff
@@ -287,8 +287,6 @@ enum {
 	 *  We will keep  this value in order to save binary compatibility.
 	 */
 	FI_PROTO_MXM,
-	FI_PROTO_IWARP_RDM,
-	FI_PROTO_IB_RDM,
```

Comment:
These values must remain, or it breaks source and binary compatibility.  This modifies the values of enums defined further down.

Reply:
Done